translate/src/modules/resourceprogress/components/ResourceProgress.tsx

```diff
@@ -1,37 +1,21 @@
 import { Localized } from '@fluent/react';
 import React, { useCallback, useState } from 'react';
 import { Link } from '~/context/location';
-import type { Stats } from '~/core/stats';
+import { Stats, useStats } from '~/core/stats';
 import { asLocaleString, useOnDiscard } from '~/core/utils';
 import { ProgressChart } from './ProgressChart';
 import './ResourceProgress.css';
 
 type Props = {
-  stats: Stats;
-};
-
-type ResourceProgressProps = {
   percent: number;
   stats: Stats;
   onDiscard: () => void;
 };
 
-function ResourceProgress({
-  percent,
-  stats,
-  onDiscard,
-}: ResourceProgressProps) {
-  const {
-    approved,
-    pretranslated,
-    warnings,
-    errors,
-    missing,
-    unreviewed,
-    total,
-  } = stats;
+function ResourceProgressDialog({ percent, stats, onDiscard }: Props) {
```

Comment:
As evidenced by your link, "drop-down" is rather strongly connected with a list of things to pick from, and e.g. the resource progress view isn't that, really.

At least to me, "panel" would be a thing that takes space in the layout, whereas the current "dialog" components are on top of the normal flow.

I ended up with "dialog" because I couldn't think of a more appropriate term. Happy to use something else, if we come up with it.

Reply:
Would you be ok with me filing an issue about harmonising the name for this thing across the codebase? That would allow going forward with this (and similar changes at least in #2518) for now, and fixing it separately.

Reply:
Yes!

Reply:
Done: #2523.